domain-services/fiscal-event-service/src/test/resources/application_test.properties

```diff
@@ -0,0 +1,7 @@
+#server.servlet.contextPath=/ifix-master-data
+#server.port=8030
+
+#spring.data.mongodb.database=test
```

Comment:
If the property is useful, then uncomment it and use it. Otherwise, if it is not going to be used, then remove it. 
This applies to all properties in all projects.

Reply:
removed.